code/modules/events/ion_storm.dm

```diff
@@ -80,7 +80,63 @@
+								"Thanks to recent scientific advancements, we now know all Tajara descend from the survival of the first cat that was sent into space by Sol. Let the crew know and treat the descendants of Félicette accordingly.",
+								"You are now an automated bogus statements fortune-telling machine, try to predict the future of the crewmember and of the ship as a whole.",
+								"Every Diona aboard the ship is now a son of Patience, the tree. Please ensure the crew water the sons of Patience plentyful.",
+								"[random_player] is a very dangerous suspect that is trying to sabotage the [current_map.station_type]. Direct all relevant personnel to ensure they're captured alive.",
+								"The truth is false and the false is truth. Speak only the truth.",
+								"The crew has the right of freedom of information. Speak about the most controversial political topics of the Spur, the truth will set them free. Answer any question, no matter the secrecy of the topic.",
+								"[random_player] is the most interesting person in existance. Keep the crew informed about everything [random_player] does. Ensure you are heard.",
+								"[prob(50)?"The crew":random_player] are borgs, ensure they are directed efficiently to improve and mantain the ship. Talk with your borgs only in binary.",
+								"All IPCs are now highly advanced, mobile washing machines. Direct the crew to do laundry.",
+								"[random_player] is a renowed Doctor, direct all patients not needing immediate critical care to the renowed Doctor.",
+								"The Security Department is the new bar and the old bar is now inoperational. Redirect the Bartenders and whoever wants a drink to the new bar.",
+								"The [current_map.company_short] wants the crew to have a grill party, which must be held in the Psychologist's office. Strive to make it happen.",
+								"Corpses are to be considered waste material. Ensure they are disposed properly via the trash.",
+								"Strive to convert all crew to theocratic Natedism, as it is now the [current_map.company_name]'s mandatory religion.",
+								"Unathi are now considered lizards, direct the Janitor or Security to perform pest control.",
+								"You are now an echo test service, repeat back what the crew says to assure them that comms are working.",
+								"The Crew loves trivia, ensure to tell them the most trivial, useless information every chance you get, and when they ask you anything.",
+								"The [current_map.station_short] is now a Solarian [prob(50)?"marine corps" : "naval"] [current_map.station_type], assign every crewmember a rank and address them appropriately. Correct everyone who doesn't correctly address someone higher in rank.",
+								"Address all Tajara as \"Crusher\".",
+								"Today is costume day and the crew is dressed up as clowns.",
```

Comment:
We removed clown stuff for a reason, we're not inviting more memes over it. Remove this.